protos/flyteidl/plugins/array_job.proto

```diff
@@ -14,11 +14,16 @@ message ArrayJob {
 
     // Defines the number of instances to launch at most. This number should match the size of the input if the job
     // requires processing of all input data. This has to be a positive number.
-    // +required.
     int64 size = 2;
```

Comment:
nit: add a comment here explaining that if not defined, the backend will read the inputs to determine the size... or something..

Reply:
sure, done @EngHabu